tests/queries/0_stateless/01086_odbc_roundtrip.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+CUR_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. $CUR_DIR/../shell_config.sh
+
+
+for i in $(seq 1 10); do
+    ${CLICKHOUSE_CLIENT} -q "select count() > 1 as ok from (select * from odbc('DSN={ClickHouse DSN (ANSI)}','system','tables'))" 2>/dev/null && break
```

Comment:
Ok but maybe also add sleep 0.1? Otherwise all ten iterations may complete too quickly.
I remember that there is a timeout... but the query may fail more soon?

Reply:
may be we can try to start bridge explicitly in advance? and wait till it will respond somthing?

Reply:
The issue happens only in the most heavy builds due to 200ms timeout.
We can start the bridge in advance in the test... but it's unreasonably to always start the bridge in advance because not everyone is using ODBC dictionaries.